Source/Scene/BillboardCollection.js

```diff
@@ -736,6 +747,16 @@ define([
             componentsPerAttribute : 3,
             componentDatatype : ComponentDatatype.FLOAT,
             usage : buffersUsage[DISTANCE_DISPLAY_CONDITION_INDEX]
+        }, {
+            index : attributeLocations.textureOffset,
+            componentsPerAttribute : 4,
+            componentDatatype : ComponentDatatype.FLOAT,
+            usage : buffersUsage[TEXTURE_OFFSET]
+        }, {
+            index : attributeLocations.dimensions,
+            componentsPerAttribute : 2,
```

Comment:
Is there anyway to pack this into one component and perhaps store as `z` in `DISTANCE_DISPLAY_CONDITION_INDEX`?  Haven't looked closely, but perhaps precision requirements are not stiff here and fewer bits can be used?

Reply:
Is there a way to pack a `Cartesian2` into one float?  I'm not quite sure how to do this

Reply:
@bagnell?

Reply:
There are no bit-wise operators in GLSL but you can shift bits by multiplying powers of 2. See any of the `writeCompressedAttrib*` functions. So if you want to pack a `Cartesian2` into a float, you'll have 12 bits for each component. Is that enough precision? It would look something like this for packing:

```javascript
var LEFT_SHIFT12 = Math.pow(2.0, 12.0);
var width = Math.floor(Math.clamp(width, 0.0, LEFT_SHIFT12));
var height = Math.floor(Math.clamp(height, 0.0, LEFT_SHIFT12));
var dimensions = width * LEFT_SHIFT12 + height;
```

Then, reverse it in the vertex shader to unpack.